include/sparrow/layout/primitive_array.hpp

```diff
@@ -176,29 +180,56 @@ namespace sparrow
         const_value_iterator value_cbegin() const;
         const_value_iterator value_cend() const;
 
-        static arrow_proxy create_proxy(size_type n);
+        static arrow_proxy create_proxy(
+            size_type n,
+            std::optional<std::string_view>&& name = std::nullopt,
+            std::optional<std::string_view>&& metadata = std::nullopt
```

Comment:
Arguments should be values instead of rvalue references.

Reply:
done